packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-oie/index.md

```diff
@@ -0,0 +1,247 @@
+
+| Function                                 | Return Type | Example                                                             | Output                    |
+| ---------------                          | ----------- | -------                                                             | -----                     |
+| `$string_object.parseCountryCode`        | CountryCode | `user.profile.country.parseCountryCode()`                           | US (CountryCode object)   |
+|                                          |             | `user.profile.countryAlpha2.parseCountryCode()`                     | US (CountryCode object)   |
+|                                          |             | `user.profile.countryAlpha3.parseCountryCode()`                     | US (CountryCode object)   |
+|                                          |             | `'840'.parseCountryCode()`                                          | US (CountryCode object)   |
+| `$country_code_object.toAlpha2`          | String      | `'USA'.parseCountryCode().toAlpha2()`                               | "US"                      |
+| `$country_code_object.toAlpha3`          | String      | `'840'.parseCountryCode().toAlpha3()`                               | "USA"                     |
+| `$country_code_object.toNumeric`         | String      | `'United States'.parseCountryCode().toNumeric()`                    | "840"                     |
+| `$country_code_object.toName`            | String      | `'US'.parseCountryCode().toName()`                                  | "United States"           |
+
+**Note:**  All these functions can be called on ISO 3166-1 2-character country codes (Alpha 2), 3-character country codes (Alpha 3), numeric country codes, and country code Strings.
+The functions return the output in the format specified by the function name.
+
+For more information on these codes, see the [ISO 3166-1 online lookup tool](https://www.iso.org/obp/ui/#search/code/).
+
+
+### Group Functions
+
+> **Note:** For the following expression examples, assume that *user* is a member of the following groups.
+
+| Group ID                 | Group Name               | Group Type            |
+| --------                 | -----------              | -----------           |
+| 00gak46y5hydV6NdM0g4     | Everyone                 | BUILT_IN              |
+| 00g1emaKYZTWRYYRRTSK     | West Coast Users         | OKTA_GROUP            |
+| 00garwpuyxHaWOkdV0g4     | West Coast Admins        | OKTA_GROUP            |
+| 00gjitX9HqABSoqTB0g3     | Engineering Users        | APP_GROUP             |
+
+Group functions take in a list of search criteria as input. Each search criteria is a key-value pair:<br>
+Key: Specifies the matching attribute. Currently supported keys are `group.id`, `group.type`, and `group.profile.name`.<br>
+Value: Specifies a list matching values which can be exact values or a regex pattern (Only start with is supported)
```

Comment:
Nit:  Specifies a list `of` matching values ...

Nit ... a regex pattern (only supporting the [wildcard] wildcard to match `starts with`)

(Or something similar.  We should explicitly spell out exactly which wildcard(s) are supported, and how they're constrained, and the phrasing could use a tweak.)